### PR TITLE
Cfn asg. Cross template Resource transferring (recordsets)

### PR DESCRIPTION
**Pull request by velychkovskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,4 +103,4 @@ This instruction provides:
 
 3. Validate cfn_asg template and Create webApp Stack:
    - aws cloudformation validate-template --template-body file://ops/cloudformation/cfn_asg.yml
-   - aws cloudformation deploy --stack-name webAppASG --template-file ops/cloudformation/cfn_asg.yml --parameter-overrides VPCStackName=${VPCStackName} PuppetScriptVersion=0.0.1 --capabilities CAPABILITY_NAMED_IAM
+   - aws cloudformation deploy --stack-name aveli-webAppASG --template-file ops/cloudformation/cfn_asg.yml --parameter-overrides VPCStackName=DevVPC PuppetScriptVersion=0.0.1 HostedZoneName=devopsa3.me.uk --capabilities CAPABILITY_NAMED_IAM
```
Comment (vecinomio) [inline README.md]: It looks like a hardcode in README. Please use variables, like in previous instructions

File: ops/cloudformation/cfn_asg.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@
 #                           --parameter-overrides                              \
 #                             VPCStackName=DevVPC | ProdVPC                    \
 #                             PuppetScriptVersion=<version>                    \
+#                             HostedZoneName=devopsa3.me.uk                    \
```
Comment (vecinomio) [inline ops/cloudformation/cfn_asg.yml]: The same. Would use variable like <hostedzonename.me.uk> or something like that.

File: README.md (hunk 1 above)
Comment (velychkovskyi) [reply README.md]: ok, the same style as previous blocks

File: ops/cloudformation/cfn_asg.yml (hunk 2 above)
Comment (velychkovskyi) [reply ops/cloudformation/cfn_asg.yml]: Yam

**Pull request merged**